microservice/src/test/java/com/skytouch/MicroserviceApplicationTests.java

```diff
@@ -0,0 +1,60 @@
+package com.skytouch;
```

Comment:
Is this class intended to contain unit or integration tests for the application?
If it's the case of integration tests, I consider you ought to add unit test classes

Reply:
I decided to create a specific test class for each service and for each repository.(ProductRepositoryTest, ProductMessageConsumerTest and ProductServiceTest )